documentation/incremental_settings.md

```diff
@@ -0,0 +1,76 @@
+# Incremental Settings for Ansible Values
+
+The general approach in Ansible is that you provide all of the values for the
```

Comment:
This is not entirely true. Ansible modules generally are using the concept outlined here (perform pointed changes and preserve everything else). The sysctl module does this and it seems to be the norm rather than an exception. That's why the interface of modules usually translates in a fairly straightforward way to the role interfaces described here. It is roles and playbooks that usually describe the complete state of the system they are managing.

Reply:
addressed in https://github.com/linux-system-roles/linux-system-roles.github.io/commit/4577d2c20cb2a14ed176486cb823442645d7088b